firebase/database.rules.json

```diff
@@ -15,6 +15,8 @@
           ||
           (auth != null  && query.orderByChild == 'status' && query.equalTo == 'build_area_tutorial' && query.limitToFirst == 1)
           ||
+          (auth != null  && query.orderByChild == 'name' && query.equalTo == 'build_area_tutorial_solid_waste' && query.limitToFirst == 1)
```

Comment:
@laurentS this is how you could query the tutorial for now... I think we need to find a better way in the future to do this. we need to make sure that we can have multiple tutorials for built area. but for now it should be fine to query like this.